history-service/src/controller/history_crud_service_handlers/get_attempts_handler.ts

```diff
@@ -30,38 +39,51 @@ class GetAttemptsHandler implements IApiHandler<GetAttemptsRequest, GetAttemptsR
       return GetAttemptsHandler.buildErrorResponse('UserId or Username must be supplied');
     }
 
+    let { userId } = request;
+    if (request.username) {
+      // Convert to user ID
+      const userObject = await super.getUser(User.create({
+        username: request.username,
+      }));
+
+      if (!userObject || !userObject.userInfo) {
+        return GetAttemptsHandler.buildErrorResponse('Username is invalid');
+      }
+
+      userId = userObject.userInfo?.userId;
+    }
+
+    console.log(`user ID: ${userId}`);
```

Comment:
I think you forgot to clear this `console.log()`

Reply:
Removed in c962730.